core/src/main/scala/kafka/tools/EndToEndLatency.scala

```diff
@@ -71,21 +71,27 @@ object EndToEndLatency {
       props
     }
 
-    val consumerProps = loadPropsWithBootstrapServers
+    val consumerProps: Properties = new Properties()
+    consumerProps.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, brokerList)
```

Comment:
`loadPropsWithBootstrapServers` adds the bootstrap servers config, so this line is not necessary.